lib/api/route/notifications.dart

```diff
@@ -1,11 +1,7 @@
 
 import '../core.dart';
 
-// This endpoint is undocumented.  Compare zulip-mobile:
-//   https://github.com/zulip/zulip-mobile/blob/86d94fa89/src/api/notifications/savePushToken.js
-// and see the server implementation:
-//   https://github.com/zulip/zulip/blob/34ceafadd/zproject/urls.py#L383
-//   https://github.com/zulip/zulip/blob/34ceafadd/zerver/views/push_notifications.py#L47
+/// https://zulip.com/api/add-fcm-token
 Future<void> registerFcmToken(ApiConnection connection, {
```

Comment:
After reading these API docs against our code, I just sent #998 to more fully update to what the docs say. I think that replaces part of this commit from this PR:
e20fa5e99 api: Add unregister{Apns,Fcm}Token; link to new docs for register-token fns